website/docs/r/servicecatalog_portfolio.html.markdown

```diff
@@ -0,0 +1,43 @@
+---
+layout: "aws"
+page_title: "AWS: aws_servicecatalog_portfolio"
+sidebar_current: "docs-aws-resource-servicecatalog-portfolio"
+description: |-
+  Provides a resource to create a Service Catalog portfolio
+---
+
+# aws\_servicecatalog\_portfolio
+
+Provides a resource to create a Service Catalog Portfolio.
+
+## Example Usage
+
+```hcl
+resource "aws_servicecatalog_portfolio" "portfolio" {
+  name = "My App Portfolio"
+  description = "List of my organizations apps"
+  provider_name = "Brett"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the portfolio.
+* `description` - (Required) Description of the portfolio
+* `provider_name` - (Required) Name of the person or organization who owns the portfolio.
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `id` - The ID of the Service Catalog Portfolio.
+
+## Import
+
+Service Catalog Portfolios can be imported using the `service catalog portfolio id`, e.g.
+
+```
+$ terraform import aws_servicecatalog_portfolio.testfolio port-12344321
```

Comment:
Can you also add a test for imports? as in: https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/import_aws_vpc_test.go